[API-692] Clean up documentation for athletes endpoints

hariniriyer opened the pull request:
Added the sample responses for the GET and PUT endpoints for the athlete.

hariniriyer commented:
@zackisaacs PTAL and let me know if I missed to stub out something.

zackisaacs commented on swagger/swagger.json.mustache:
```diff
@@ -99,6 +99,55 @@
+            },
+            "examples": {
+                "application/json": {
+                     "id": 12345,
```
This is a real athlete. Let's put a bigger ID here that will never correspond to a real athlete, like `12345678987654321`. Any number several times bigger than the population of the world should do 😉

zackisaacs commented on swagger/swagger.json.mustache:
```diff
@@ -99,6 +99,55 @@
+                     "created_at": "2017-11-14T02:30:05Z",
+                     "updated_at": "2018-02-06T19:32:20Z",
+                     "badge_type_id": 4,
+                     "profile_medium": "https://xxxxxx.cloudfront.net/pictures/athletes/12345/67890/2/medium.jpg",
```
Let's stub out the IDs in the photos here and in the line below, e.g.
```
"profile_medium": "https://xxxxxx.cloudfront.net/pictures/athletes/123456789/123456789/2/medium.jpg"
```

zackisaacs commented on swagger/swagger.json.mustache:
```diff
@@ -99,6 +99,55 @@
+                     "weight": 0,
+                     "bikes": [
+                     {
+                        "id": "b241",
```
Let's keep the IDs to be strictly numbers, but just make sure they're very large.

hariniriyer replied:
This is how these are stored on our end. This is a stubbed out id.

zackisaacs commented on swagger/swagger.json.mustache:
```diff
@@ -99,6 +99,55 @@
+                     ],
+                     "shoes": [
+                     {
+                        "id": "g2106",
```
Let's make this a large number.

hariniriyer replied:
This is how these are stored on our end. This is a stubbed out id.

zackisaacs commented on swagger/swagger.json.mustache:
```diff
@@ -133,6 +182,55 @@
+            },
+            "examples": {
+                "application/json": {
+                    "id": 12345,
```
`12345678987654321`

zackisaacs commented on swagger/swagger.json.mustache:
```diff
@@ -133,6 +182,55 @@
+                    "updated_at": "2018-02-06T19:32:20Z",
+                    "badge_type_id": 4,
+                    "profile_medium": "https://xxxxxx.cloudfront.net/pictures/athletes/12345/67890/2/medium.jpg",
+                    "profile": "https://xxxxx.cloudfront.net/pictures/athletes/12345/67890/2/large.jpg",
```
Same thing here -- let's sanitize the IDs in these pictures.

zackisaacs commented on swagger/swagger.json.mustache:
```diff
@@ -133,6 +182,55 @@
+                    "weight": 0,
+                    "bikes": [
+                    {
+                        "id": "b241",
```
Let's make this a number.

zackisaacs commented on swagger/swagger.json.mustache:
```diff
@@ -133,6 +182,55 @@
+                    ],
+                    "shoes": [
+                    {
+                       "id": "g2106",
```
Let's make this a number.

hariniriyer commented:
@zackisaacs PTAL

zackisaacs commented on swagger/swagger.json.mustache:
```diff
@@ -91,16 +91,65 @@
-        "summary": "Get Logged-in Athlete",
-        "description": "Returns the currently logged-in athlete",
+        "summary": "Get Authenticated Athlete",
+        "description": "Returns the currently authenticated athlete",
```
nit: I believe we decided on periods for descriptions 😉

zackisaacs commented on swagger/swagger.json.mustache:
```diff
@@ -114,7 +163,7 @@
         "operationId": "updateLoggedInAthlete",
         "summary": "Update Athlete",
-        "description": "Update the currently logged-in athlete",
+        "description": "Update the currently authenticated athlete",
```
Period.

zackisaacs commented on swagger/swagger.json.mustache:
```diff
@@ -131,9 +180,58 @@
         "responses": {
           "200": {
-            "description": "Profile information for the logged-in athlete",
+            "description": "Profile information for the authenticated athlete",
```
Period.

hariniriyer commented:
@MateoAO @zackisaacs I changed the name to Marianne V and I changed all the descriptions in the file to have a period at the end (the ones that did not). PTAL if you want. I will merge it tomorrow.

MateoAO commented:
👍 don't forget to squash
